src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__container">
+        <div class="product-card__product-image">
+          <img
+            src="./images/imac.jpeg"
+            alt="apple monitor"
+            class="product-card__product-card-image"
```

Comment:
No reason to repeat "product-card" again, instead try "product-card__product-image" for better readability

Reply:
Also, alt text is incorrect, in line with the checklist's bad example:

STILL BAD example:
```html
<img alt="phone" />
```
GOOD example:
```html
<img alt="Samsung Galaxy S22 2022 8/128GB Green" />
```

Instead, try putting the model of the product, and side note is that it's an all-in-one workstation rather than a monitor

Reply:
You suggest to change child class name to parent class name.